clr-installer/main.go

```diff
@@ -228,7 +228,7 @@ func execute(options args.Args) error {
 			return err
 		}
 		options.CfDownloaded = true
-	} else {
+	} else if ok, err := utils.FileExists(options.ConfigFile); !ok || err != nil {
 		return errors.Errorf("No valid configuration file")
```

Comment:
Perhaps we should use the same or similar messages the file local file is not there?
```suggestion
		return errors.Errorf("Cannot acesss configuration file %q: %s", options.ConfigFile, err)
```

Reply:
I can do:
`return errors.Errorf("Cannot acesss configuration file %q", options.ConfigFile)`

The error will typically be `nil` when the file doesn't exist. It seems strange to print that.